ddtrace/tracer/span.go

```diff
@@ -337,6 +337,24 @@ func (s *span) String() string {
 	return strings.Join(lines, "\n")
 }
 
+// Format implements fmt.Formatter.
+func (s *span) Format(f fmt.State, c rune) {
+	switch c {
+	case 's':
+		fmt.Fprint(f, s.String())
+	case 'v':
+		fmt.Fprintf(f, "dd.trace_id=%d dd.span_id=%d dd.service=%s", s.TraceID, s.SpanID, s.Service)
```

Comment:
we should have `dd.trace_id=%d dd.span_id=%d` be last in the list here.

this makes it easier/more performant for the grok parsing rule.